new-website/src/components/location/location-table.jsx

```diff
@@ -2,51 +2,93 @@ import React, { useState } from "react"
 import firebase from "gatsby-plugin-firebase"
 import MaterialTable from "material-table"
 import { LOCATION_COLLECTION } from "."
-import { useIntl } from "gatsby-plugin-intl"
+import { navigate, useIntl } from "gatsby-plugin-intl"
 import { generateI18N } from "./table-i18n"
 import { Roller } from "react-spinners-css"
+import shortid from "shortid"
 
-function choosePartnerId(entryPartnerId, userPartnerIds = []) {
-    if (entryPartnerId && userPartnerIds.includes(entryPartnerId)) {
-        return entryPartnerId
+function choosePartnerId(entryPartnerId, userPartnerIds, onError) {
+    if (entryPartnerId) {
+        if (userPartnerIds.includes(entryPartnerId)) {
+            return entryPartnerId
+        }
+        // TODO: Issue this error to sentry?
+        onError({ code: "choosePartnerId/inaccessible" })
+        throw new Error("Provided partner ID inaccessible for user")
+    } else {
+        // TODO: Depending on the context we may want users to choose and not just
+        // take the first in the sequence.
+        if (userPartnerIds.length > 0) {
+            return userPartnerIds[0]
+        }
+        // TODO: Issue this error to sentry?
```

Comment:
I assume this is one of the errors we want to see in Sentry right? How would we do this?

Reply:
Yes 👍

Reply:
How would we do it? Can you drop me a minimal snippet?

Reply:
checkout the 404.jsx page

Reply:
Done, but not sure it works. See: https://github.com/ImmunHelden/ImmunHelden.de/pull/155#discussion_r451628297